cogboard-webapp/src/components/Widget/index.js

```diff
@@ -121,6 +121,8 @@ const Widget = ({ id, index }) => {
   return (
     <>
       <StyledCard
+        showShadow={expanded}
+        showBorder={isAuthenticated && 'WhiteSpaceWidget' === type}
```

Comment:
Move this logic to const and type === 'WhiteSpaceWidget' seems more natural for me.

Reply:
done